proto/waste.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "types/info.proto";
+
+// WasteApi exposes details about units of waste that have been produced by the building
+service WasteApi {
+  rpc ListWasteRecords(ListWasteRecordsRequest) returns (ListWasteRecordsResponse);
+  rpc PullWasteRecords(PullWasteRecordsRequest) returns (stream PullWasteRecordsResponse);
+}
+
+service WasteInfo {
+  rpc DescribeWasteRecord(DescribeWasteRecordRequest) returns (WasteRecordSupport);
+}
+
+message WasteRecord {
+  // The id of the waste record assigned by the external waste management system
+  string id = 1;
+  // The time the record was created, this is not the time the waste was created
+  google.protobuf.Timestamp recordCreatedTime = 2;
+  // The weight of the waste. Use the InfoApi to find the actual unit the device uses.
+  float weight = 3;
+  // The system the waste was collected from
+  string system = 4;
```

Comment:
Can you document whether the intent is for this to be presented to people or used for identity?